vagrant/ansible/roles/client.test/tasks/main.yml

```diff
@@ -0,0 +1,19 @@
+---
+- name: Fetch the latest samba-integration-tests repo
+  git:
+    repo: 'https://github.com/spuiuk/samba-integration.git'
+    version: tests
+    dest: /root/samba-integration-tests
```

Comment:
shouldn't use `/root`

Reply:
This is within the client vm which is created by scratch and the tests have to be run as root. I don't understand why we cannot use absolute paths here.